pkg/langruntime/langruntime.go

```diff
@@ -235,8 +235,7 @@ func (l *Langruntimes) GetBuildContainer(runtime, depsChecksum string, env []v1.
 			"cd $GOPATH/src/kubeless",
 			"dep ensure > /dev/termination-log 2>&1")
 	case strings.Contains(runtime, "dotnetcore"):
-		command = appendToCommand(command,
-			"dotnet restore "+installVolume.MountPath+" --packages "+installVolume.MountPath+"/packages")
+		return v1.Container{}, nil
```

Comment:
Returning a default object.

Reply:
I would throw an error here saying that the dependencies file is not needed. Something like:
```
return v1.Container{}, fmt.Errorf(".NET runtime does not require a dependencies file")
```

Reply:
@andresmgot , about throwing an error, would it be more complicated to handled that? Today, if I throw an error like this, currently it would fail this way, due the existent pipeline:
```bash
time="2018-06-05T00:49:41Z" level=info msg="Processing change to Function default/hey" pkg=function-controller
time="2018-06-05T00:49:42Z" level=error msg="Function can not be created/updated: dotnetcore does not require a dependencies file" pkg=function-controller
time="2018-06-05T00:49:42Z" level=error msg="Error processing default/hey (giving up): dotnetcore does not require a dependencies file" pkg=function-controller
E0605 00:49:42.737356       1 function_controller.go:185] dotnetcore does not require a dependencies file
```

Reply:
Fair enough. Probably the best solution would be to disable this step at the configuration level but we can live with this solution for the moment. Can you at least add a warning message?

```
logrus.Warn("dotnetcore does not require a dependencies file")
```

Reply:
Agreed (y), I will add that.